src/main/java/com/google/step/snippet/external/W3SchoolClient.java

```diff
@@ -6,12 +6,30 @@
 import org.jsoup.nodes.Document;
 import org.jsoup.select.Elements;
 
-public final class W3SchoolClient {
+public final class W3SchoolClient implements Client {
```

Comment:
Didn't notice this before, but the website is W3Schools, not W3School. Can that be changed all around?

Reply:
changed